chore: updating dependabot to support gha, python, docker and dev container packages

rajpalc7 opened the pull request.

tra371 committed:
Add support for vc_di credential issue

Co-authored-by: Orjiene Kenechukwu <orjienekenechukwu@gmail.com>

Signed-off-by: Sarthak Vijayvergiya <sarthakvijayvergiya@gmail.com>

sarthakvijayvergiya committed:
Copy code & cleanup

Signed-off-by: Sarthak Vijayvergiya <sarthakvijayvergiya@gmail.com>

sarthakvijayvergiya committed:
Fixed test_routes.py

Signed-off-by: Sarthak Vijayvergiya <sarthakvijayvergiya@gmail.com>

sarthakvijayvergiya committed:
Update vc_di test cases

Signed-off-by: Sarthak Vijayvergiya <sarthakvijayvergiya@gmail.com>

sarthakvijayvergiya committed:
fix linting

Signed-off-by: Sarthak Vijayvergiya <sarthakvijayvergiya@gmail.com>

sarthakvijayvergiya committed:
linting & code formatting

Signed-off-by: Sarthak Vijayvergiya <sarthakvijayvergiya@gmail.com>

sarthakvijayvergiya committed:
linting

Signed-off-by: Sarthak Vijayvergiya <sarthakvijayvergiya@gmail.com>

sarthakvijayvergiya committed:
fix test_create_credential_vcdi

Signed-off-by: Sarthak Vijayvergiya <sarthakvijayvergiya@gmail.com>

sarthakvijayvergiya committed:
fix store cred

Signed-off-by: Sarthak Vijayvergiya <sarthakvijayvergiya@gmail.com>

supersonicwisd1 committed:
Fix issuance date & proof web request

Signed-off-by: Sarthak Vijayvergiya <sarthakvijayvergiya@gmail.com>

sarthakvijayvergiya committed:
resolve comments & restructure vc_di models

Signed-off-by: Sarthak Vijayvergiya <sarthakvijayvergiya@gmail.com>

sarthakvijayvergiya committed:
update todo comments

Signed-off-by: Sarthak Vijayvergiya <sarthakvijayvergiya@gmail.com>

ianco committed:
Allow option to switch cred type in demo and integration tests

Signed-off-by: Ian Costanzo <ian@anon-solutions.ca>

sarthakvijayvergiya committed:
Merge pull request #35 from ianco/feat/w3c-implementation

Allow option to switch cred type in demo and integration tests

sarthakvijayvergiya committed:
Fix import & added proof legacy request

Signed-off-by: Sarthak Vijayvergiya <sarthakvijayvergiya@gmail.com>

sarthakvijayvergiya committed:
Merge branch 'main' into feat/vc-di

sarthakvijayvergiya committed:
Merge branch 'main' into feat/vc-di

sarthakvijayvergiya committed:
fix comments

Signed-off-by: Sarthak Vijayvergiya <sarthakvijayvergiya@gmail.com>

rajpalc7 commented:
Hi @WadeBarnes , this is ready for review now.

ianco committed:
Merge branch 'main' into feat/vc-di

sarthakvijayvergiya committed:
fix entropy validation

Signed-off-by: Sarthak Vijayvergiya <sarthakvijayvergiya@gmail.com>

sarthakvijayvergiya committed:
fix test

Signed-off-by: Sarthak Vijayvergiya <sarthakvijayvergiya@gmail.com>

WadeBarnes reviewed (changes requested):
@rajpalc7, You are missing a number of Docker files.

rajpalc7 committed:
chore: updating dependabot to support gha, python, docker and dev container packages

Signed-off-by: Rajpal Chauhan <rajpal.chauhan@gov.bc.ca>

rajpalc7 commented:
Hi @WadeBarnes,  other Docker files are added now.

dependabot[bot] committed:
chore(deps): Bump ecdsa in the pip group across 1 directory

Bumps the pip group with 1 update in the / directory: [ecdsa](https://github.com/tlsfuzzer/python-ecdsa).


Updates `ecdsa` from 0.16.1 to 0.19.0
- [Release notes](https://github.com/tlsfuzzer/python-ecdsa/releases)
- [Changelog](https://github.com/tlsfuzzer/python-ecdsa/blob/master/NEWS)
- [Commits](https://github.com/tlsfuzzer/python-ecdsa/compare/python-ecdsa-0.16.1...python-ecdsa-0.19.0)

---
updated-dependencies:
- dependency-name: ecdsa
  dependency-type: direct:production
  dependency-group: pip
...

Signed-off-by: dependabot[bot] <support@github.com>

sarthakvijayvergiya committed:
fix vc_di-issued credential proof request

Signed-off-by: Sarthak Vijayvergiya <sarthakvijayvergiya@gmail.com>

swcurran committed:
Merge pull request #2933 from hyperledger/dependabot/pip/pip-614c5aae4a

chore(deps): Bump ecdsa from 0.16.1 to 0.19.0 in the pip group across 1 directory

ianco committed:
Merge branch 'main' into feat/vc-di

ianco committed:
Merge pull request #2861 from Whats-Cookin/feat/vc-di

feat: Integrate AnonCreds with W3C VCDI Format Support in ACA-Py

jamshale committed:
Upgrade to anoncreds via api endpoint (#2922)

* Add upgrade to anoncreds via api

Signed-off-by: jamshale <jamiehalebc@gmail.com>

---------

Signed-off-by: jamshale <jamiehalebc@gmail.com>

esune committed:
Feature: use decorators for admin api authentication (#2860)

* Add authentication decorators for admin API

Signed-off-by: Emiliano Suñé <emiliano.sune@gmail.com>

---------

Signed-off-by: Emiliano Suñé <emiliano.sune@gmail.com>

swcurran committed:
Merge branch 'main' into dependabot
